PowerShellTools.HostService/ServiceManagement/Debugging/PowerShellDebuggingService.cs

```diff
@@ -95,6 +107,8 @@ public PowerShellDebuggingService()
             _psBreakpointTable = new HashSet<PowerShellBreakpointRecord>();
             _debugOutput = true;
             _installedPowerShellVersion = DependencyUtilities.GetInstalledPowerShellVersion();
+            _validator = new PowerShellDebuggingServiceAttachValidator(this);
+            _forceStop = false;

```

Comment:
This seems to be the only occurrence of setting this flag to false, which does not look correct to me, as this constructor will only be called *for once* at the moment of host process initializes, which means during the debugging, if use ever did "forcestop" once, the entire rest life of current VS session will be treated as "forcestop".